src/utils.c

```diff
@@ -349,6 +352,18 @@ pu_is_drive(const gchar *device)
                                 output, G_REGEX_MULTILINE, 0);
 }
 
+gboolean
+pu_wait_for_partitions(GError **error)
+{
+    g_autofree gchar *udevadm_cmd = NULL;
+    udevadm_cmd = g_strdup_printf("udevadm settle --timeout %d", UDEVADM_SETTLE_TIMEOUT);
+
+    if (!pu_spawn_command_line_sync(udevadm_cmd, error))
+        return FALSE;
```

Comment:
Why did you remove the check for the exit status here?

Reply:
Since this function is now in utils, I switched to using `pu_spawn_command_line_sync()` which already checks for the exit status.

Reply:
Ah, okay. I overlooked that. :slightly_smiling_face: